launchpad-to-stargate/tutorial/02-module-migration.md

```diff
@@ -0,0 +1,624 @@
+---
+order: 2
+---
+# Module migration
+
+In this section, we will be migrating the `pofe` module from our Launchpad application. 
```

Comment:
Please explain a bit more what we will focus on in the coming chapter. What are the biggest changes going from Launchpad to Stargate? What will I be able to just copy over vs what has to change?